packages/workbox-webpack-plugin/src/lib/get-webpack-manifest-entries.js

```diff
@@ -0,0 +1,76 @@
+const url = require('url');
+
+const entry = (revision, filePath, publicPath) => ({
+  revision,
+  url: url.resolve(publicPath, filePath),
+});
+
+/**
+ * Filter chunks that have one of the provided names (from config.chunks)
+ * @param {Array<Object>} chunks Webpack chunks
+ * @param {Array<string>} include Chunk names to include
+ * @param {Array<string>} exclude Chunk names to exclude
+ * @return {Array<Object>} Filtered array of chunks
+ */
+const filterChunks = (chunks, include, exclude) =>
+  chunks.filter((chunk) =>
+    // chunk has a name
+    Object.prototype.hasOwnProperty.call(chunk, 'name')
+      // did not specify include or chunk in include
+      && (!Array.isArray(include) || !!~include.indexOf(chunk.name))
+      // did not specify exclude or chunk not in exclude
+      && (!Array.isArray(exclude) || !~exclude.indexOf(chunk.name))
+  );
+
+/**
+ * Maps webpack asset filenames to their chunk hash
+ * @param {Array<Object>} chunks Webpack chunks
+ * @return {Object} {filename: hash} map
+ */
+const mapAssetsToChunkHash = (chunks) =>
+  chunks.reduce((assetMap, chunk) =>
+      Object.assign(
+        assetMap,
+        ...chunk.files.map((f) => ({[f]: chunk.renderedHash}))
+    ), {});
+
+const generateManifestEntries = (compiler, compilation, config) => {
+  const {hash} = compilation; // compilation hash
```

Comment:
You could inline the single use of hash to the bottom of this function as `compilation.hash`.

Reply:
sure!

Reply:
Thanks!